prototype_lesson22/index.js

```diff
@@ -0,0 +1,90 @@
+const universityPrototype = {
+  universityName: "Polytechnic",
+  dean: "Taras Samoilenko",
+};
+
+const university = Object.create(universityPrototype);
+
+const facultyPrototype = {
+  facultyName: "IT-Technology",
+  groups: [],
+  enlistStudent(studentName) {
+    if (this.groups.length < 12) {
+      this.groups.push([studentName]);
+      console.log(`Зараховано студента ${studentName}`);
+    } else {
+      console.log(
+        `Вибачте ця група вже заповнена ми не можемо зарахувати студента ${studentName}`
```

Comment:
groups is arr of arr [[]]
if group is filled create another
in other words
```
const group = ['Taras', 'Ostap']
const groups = [group]
```

Reply:
resolved